### ISSUE TITLE
Consider whitelisting virtual meeting URLs

### ISSUE THREAD

**Issue by tonydiep** [Eloston/disable-html5-autoplay#228]
Thanks for this excellent extension!

You might want to consider whitelisting meet.google.com, teams.microsoft.com, and other virtual meeting URLs.

Took me a long time to troubleshoot virtual meeting video not working. It's not obvious when using Teams and Google Meet that you have to whitelist HTML5 video.

**Comment by Adambean** [comment]
To name a few Google ones:

chat.google.com
hangouts.google.com
meet.google.com

**Comment by vivian-ng** [comment]
Yes, second this request. It took a while for me to figure out that this plugin was causing my audio/video problems with Google Meet.

**Comment by AliGeyik** [comment]
This Chrome extension breaks Chrome Webcam to show up at first, when you disable/re-enable the webcam, webcam capture shows up on your screen, however, you can see your voice is passing through your mic but you don't hear the speaker test sound or any other sound at all in Google Meet.
--
AliGeyik.com
